modules/library/src/main/scala/zio/elasticsearch/HttpElasticExecutor.scala

```diff
@@ -120,6 +121,36 @@ private[elasticsearch] final class HttpElasticExecutor private (esConfig: Elasti
     }
   }
 
+  private def executeCount(r: Count): Task[Int] = {
+    val req = r.query match {
+      case Some(query) =>
+        request
+          .get(uri"${esConfig.uri}/${r.index}/$Count".withParams(getQueryParams(List(("routing", r.routing)))))
+          .contentType(ApplicationJson)
+          .body(query.toJson)
+          .response(asJson[ElasticCountResponse])
+      case None =>
+        request
+          .get(uri"${esConfig.uri}/${r.index}/$Count".withParams(getQueryParams(List(("routing", r.routing)))))
+          .contentType(ApplicationJson)
+          .response(asJson[ElasticCountResponse])
+    }
+
+    sendRequestWithCustomResponse(req).flatMap { response =>
+      response.code match {
+        case HttpOk =>
+          response.body
+            .map(res => res.count)
```

Comment:
Can we use `_.count` here?

Reply:
Is `count` `Int` or `Long` value?

Reply:
It is not specified, I suppose it can be both.